cg/store/api/findbusinessdata.py

```diff
@@ -166,6 +166,31 @@ def family_samples(self, family_id: str) -> List[models.FamilySample]:
             .all()
         )
 
+    def sample_cases(self, sample_id: str) -> List[models.FamilySample]:
+        """Find the cases associated with a sample."""
+        return (
+            self.FamilySample.query.join(models.FamilySample.family, models.FamilySample.sample)
+            .filter(models.Sample.internal_id == sample_id)
+            .all()
+        )
+
+    def get_cases_from_sample(self, sample_entry_id: str) -> List[models.FamilySample]:
+        """Find cases related to a given sample."""
+        return (
+            self.FamilySample.query.join(models.FamilySample.family, models.FamilySample.sample)

```

Comment:
If this was a function returning the query you could unit test the query and also reuse it in both:
`get_cases_from_sample` and `get_sample_cases`